tests/platform_tests/api/test_sfp.py

```diff
@@ -262,7 +262,7 @@ def is_xcvr_optical(self, xcvr_info_dict):
             spec_compliance_dict = ast.literal_eval(xcvr_info_dict["specification_compliance"])
             if xcvr_info_dict["type_abbrv_name"] == "SFP":
                 compliance_code = spec_compliance_dict.get("SFP+CableTechnology")
-                if compliance_code == "Passive Cable":
+                if compliance_code in ["Passive Cable", "Unknown"]:
```

Comment:
"unknown" is not a valid compliance code. Can you test it with valid optics which follows SFF8472? There can be only two compliance code as shown below:-

<img width="536" alt="image" src="https://user-images.githubusercontent.com/45705344/221485898-10ca8ffe-5117-4699-bc18-7c5db5ee3efe.png">

Reply:
Also, wouldn't this function now return False for SFF-8472 since compliance_code would be "Unknown" and hence, we will eventually skip many SFP related tests which are meant for fiber based transceivers?